flexbox/src/androidTest/res/layout/activity_align_items_baseline_wrap_content.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright 2017 Google Inc. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  -->
+<com.google.android.flexbox.FlexboxLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/flexbox_layout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:flexDirection="row"
+    app:flexWrap="wrap"
+    app:alignItems="baseline">
+
+    <TextView
+        android:id="@+id/text1"
+        android:layout_width="30dp"
+        android:layout_height="80dp"
+        android:text="1"
+        android:gravity="bottom" />
+
+    <TextView
+        android:id="@+id/text2"
+        android:layout_width="30dp"
```

Comment:
As `30dp` and `80dp` are used three times here, I think that good idea would be to extract them to `dimen` variables.